Public/Src/Engine/Dll/DirectoryScrubber.cs

```diff
@@ -408,7 +408,6 @@ private bool TryDeleteFile(LoggingContext loggingContext, string path)
             try
             {
                 FileUtilities.DeleteFile(path, waitUntilDeletionFinished: true, tempDirectoryCleaner: m_tempDirectoryCleaner);
-                Tracing.Logger.Log.ScrubbingFile(loggingContext, path);

```

Comment:
instead, please add a `bool` argument to this method which denotes whether or not to log deleted file; then only suppress logging when deleting sideband files themselves, but please don't suppress logging which shared opaque outputs got deleted (but feel free to redirect those logs into a separate log file)